### PR TITLE
Update vetKD instructions & fix install page redirect

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/integrations/vetkeys/using-vetkeys.md [hunk 1]
```diff
@@ -19,33 +19,59 @@ This demo uses files found in [this repository](https://github.com/dfinity/examp
 git clone https://github.com/dfinity/examples/
 ```
 
-Then navigate into the directory specifically for this project:
+Then navigate into the directory specifically for this project. For the Rust implenentation, use the command:
```
Comment (yvonneanne) [inline docs/developer-docs/integrations/vetkeys/using-vetkeys.md:22]:
```suggestion
Then navigate into the directory specifically for this project. For the Rust implementation, use the command:
```

**Comment by yvonneanne** [comment]
Hi Jessie
Thanks for fixing the problems!
Note that on https://internetcomputer.org/docs/current/samples/encrypted-notes, Step 1 tells the reader to go to examples/rust/encrypted-notes-dapp for rust. This is wrong, as that directory is empty except for the readme. The same holds for the vetkd-encrypted-notes sample described on https://internetcomputer.org/docs/current/samples/vetkd-encrypted-notes.
On the later, link to the SDK installation page is wrong and export BUILD_ENV=rust must be set as well. .

File: docs/samples/encrypted-notes.md [hunk 2]
```diff
@@ -86,38 +84,22 @@ Once authenticated with II:
...
-
-For Rust deployment run:
-
-```
-export BUILD_ENV=rust
-```
-
-:::info
-Building the Rust canister requires either the Rust toolchain installed on your system or Docker-backed deployment (see below).
-:::
```
Comment (yvonneanne) [inline docs/samples/encrypted-notes.md:120]: why do you suggest to delete the Rust deployment part? The code is in the samples repo and works fine, afaics...

File: docs/samples/vetkd-encrypted-notes.md [hunk 3]
```diff
@@ -21,24 +19,17 @@ This example uses an [**insecure** implementation](https://github.com/dfinity/ex
 ## Prerequisites
 This example requires an installation of:
 
-- [x] Install the [IC SDK](https://internetcomputer.org/docs/current/developer-docs/setup/install/index.mdx).
+- [x] Install the [IC SDK](https://internetcomputer.org/docs/current/developer-docs/setup/install).
 - [x] Install `node.js` to build the web frontend.
 - [x] Download the following profile files from GitHub: https://github.com/dfinity/examples/
-- [x] If deploying in Rust, run `rustup target add wasm32-unknown-unknown` before getting started.
```
Comment (yvonneanne) [inline docs/samples/vetkd-encrypted-notes.md:27]: why do you remove the Rust instructions? They work fine after running 
`export BUILD_ENV=rust`

File: docs/developer-docs/integrations/vetkeys/using-vetkeys.md [hunk 4]
```diff
@@ -22,30 +22,42 @@ git clone https://github.com/dfinity/examples/
 Then navigate into the directory specifically for this project:
 
 ```
-cd examples/rust/vetkd
+cd examples/motoko/vetkd
```
Comment (yvonneanne) [inline docs/developer-docs/integrations/vetkeys/using-vetkeys.md:25]: both folders exist and contain code:

https://github.com/dfinity/examples/tree/master/rust/vetkd
https://github.com/dfinity/examples/tree/master/motoko/vetkd

the reader change choose which one to run

**Review by yvonneanne — APPROVED** [review]
Thanks for resolving all comments!

**Review by dprats — APPROVED** [review]

**Pull request merged**